sox.dialog.js

```diff
@@ -220,12 +208,7 @@
         const $icon = $(this).find('i');
 
         const checked = $icon.hasClass('fas');
-
-        if (checked) {
-          $icon.removeClass('fas').addClass('far');
-        } else {
-          $icon.removeClass('far').addClass('fas');
-        }
+        checked ? $icon.removeClass('fas').addClass('far') : $icon.removeClass('far').addClass('fas');
```

Comment:
I'd prefer to leave this as an explicit if rather than using the ternary, there's no real advantage of it here!